src/preferences/effectmanifesttablemodel.cpp

```diff
@@ -146,8 +146,17 @@ Qt::DropActions EffectManifestTableModel::supportedDropActions() const {
 }
 
 bool EffectManifestTableModel::removeRows(int row, int count, const QModelIndex& parent) {
-    Q_UNUSED(parent);
-    beginRemoveRows(QModelIndex(), row, row + count - 1);
+    if (!count) {
```

Comment:
implicit conversion to `int` is often a bug and is usually forbidden with `-Wconversion`. Please be explicit. 
```suggestion
    if (count == 0) {
```